language/nl/email/mention_mail.txt

```diff
@@ -0,0 +1,10 @@
+Subject: Je bent genoemd
+
+Hallo,
+
+Je bent genoemd door "{USERNAME}" in "{TOPIC_TITLE}" at "{SITENAME}".
+
+Om het onderwerp te zien waarin je genoemd bent, gebruik dan deze link:
+{U_LINK_TO_TOPIC}
```

Comment:
Please keep this in line with the offical dutch translation; https://github.com/phpbb-nl-vertaling/phpbb_nl/blob/master/language/nl/email/topic_notify.txt#L10